src/control/server/config/server_test.go

```diff
@@ -629,7 +635,7 @@ func TestServerConfig_Parsing(t *testing.T) {
 								WithBdevDeviceList(MockPCIAddr(1), MockPCIAddr(1)),
 						))
 			},
-			expValidateErr: errors.New("bdev_list contains duplicate pci"),
+			expValidateErr: errors.New("valid PCI addresses"),
```

Comment:
I'm inclined to think the old message about duplicates is more useful

Reply:
The error changed here because we're no longer validating the bdev_list entries in `BdevConfig.Validate()` -- at this point we're just checking that if the config specifies nvme, then the list of block devices should not be empty. The test is now a little awkward because it's using the `.WithBdevDeviceList()` helpers to construct the config before testing it. As these helpers cannot return an error, what we get with duplicate devices added via the helper is an empty device list. I see this as an edge case test rather than something that will be commonly hit by end users.

To put it another way, if you put duplicate addresses in your server yaml's bdev_list, you'll still see an error like this: `bdev_list: duplicate PCI address 0000:81:00.0`.